api_v3/services/AnalyticsService.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * api for getting analytics data
+ * @service analytics
+ * @package api
+ * @subpackage services
+ */
+class AnalyticsService extends KalturaBaseService
+{
+	/**
+	 * report query action allows to get a analytics data for specific query dimensions, metrics and filters.
+	 *
+	 * @action query
+	 * @param int $from query start time (in unix time)
+	 * @param int $to query end time (in unix time)
+	 * @param string $metrics comma separated metrics list
+	 * @param string $dimensions comma separated dimensions list
+	 * @param KalturaReportFilterArray $filters array of filters
+	 * @return KalturaReportResponse
+	 */
+	public function queryAction($from, $to, $metrics, $dimensions = null, $filters = null)
```

Comment:
Use one object that contains all the attributes, we may need to add more attributes in the future.